amazonec2/Module.md

```diff
@@ -37,23 +37,85 @@ First, import the `wso2/amazonec2` module into the Ballerina project.
 ```ballerina
 import wso2/amazonec2;
 ```
-The Amazon EC2 connector can be instantiated using the accessKeyId, secretAccessKey and region,
+The Amazon EC2 connector can be instantiated using the accessKeyId, secretAccessKey, securityToken and region,
 in the Amazon EC2 client config.
 
-**Obtaining Access Keys to Run the Sample**
+**Obtaining AWS credentials to Run the Sample**
 
- 1. Create a amazon account by visiting <https://aws.amazon.com/ec2/>
- 2. Obtain the following parameters
-   * Access key ID.
-   * Secret access key.
-   * Desired Server region.
+## Signing Up for AWS
+
+1. Navigate to this link <https://aws.amazon.com/>, and then choose Create an AWS Account.
+
+   **Note:** If you previously signed in to the AWS Management Console using AWS account root user credentials, choose Sign in to a different account. If you previously signed in to the console using IAM credentials, choose Sign-in using root account credentials. Then choose Create a new AWS account.
+2. Follow the online instructions - Part of the sign-up procedure involves receiving a phone call and entering a verification code using the phone keypad. AWS will notify you by email when your account is active and available for you to use.
+
+You can follow one of the below explained ways to obtain AWS credentials.
+
+### Obtaining user credentials
+
+You can access the Amazon EC2 service using the root user credentials but these credentials allow full access to all resources in the account as you can't restrict permission for root user credentials. If you want to restrict certain resources and allow controlled access to AWS services then you can create IAM(Identity and Access Management) users in your AWS account. In that case :
+
+1. Follow the steps below to get an AWS Access Key for your AWS root account:
+
+    * Go to the AWS Management Console.
+    * Hover over your company name in the right top menu and click "My Security Credentials".
+    * Scroll to the "Access Keys" section.
+    * Click on "Create New Access Key".
+    * Copy both the Access Key ID (YOUR_AMAZON_EC2_KEY) and Secret Access Key (YOUR_AMAZON_EC2_SECRET).
+
+2. Follow the steps below to get an AWS Access Key for an IAM user account:
+
+    * Sign in to the AWS Management Console and open the IAM console.
+    * In the navigation pane, choose Users.
+    * Add a check mark next to the name of the desired user, and then choose User Actions from the top.
+    * Click on Manage Access Keys.
+    * Click on Create Access Key.
+    * Click on Show User Security Credentials. Copy and paste the Access Key ID and Secret Access Key values, or click on Download Credentials to download the credentials in a CSV (file).
```

Comment:
```suggestion
    * Click **Show User Security Credentials**. Copy and paste the Access Key ID and Secret Access Key values or click **Download Credentials** to download the credentials as a CSV (file).
```
Shall we make this another step - "Copy and paste the Access Key ID and Secret Access Key values, or click on Download Credentials to download the credentials in a CSV (file)."?